examples/atlas_segmentation.ipynb

```diff
@@ -0,0 +1,440 @@
+    "function. This produces a visualisation was well as a pandas DataFrame with the quantitative\n",
+    "metrics."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, df_sas = contour_comparison(\n",
+    "    img = img_ct_test,\n",
+    "    contour_dict_a = contours_test,\n",
+    "    contour_dict_b = contours_atlas_reg_dir,\n",
+    "    contour_label_a = \"MANUAL\",\n",
+    "    contour_label_b = \"AUTOMATIC\",\n",
+    "    title='Atlas-based Segmentation',\n",
+    "    subtitle='',\n",
+    "    subsubtitle='',\n",
+    ")\n",
+    "\n",
+    "df_sas"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Multi-atlas Segmentation\n",
+    "\n",
+    "We can often improve the quality of atlas-based segmentation by using multiple atlas cases. PlatiPy\n",
+    "has built-in functionality to perform multi-atlas segmentation. The following example uses 4 atlas\n",
+    "cases to auto-segment our test case."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# make a copy of the default settings\n",
+    "user_settings = MUTLIATLAS_SETTINGS_DEFAULTS\n",
+    "\n",
+    "# Define the list of structures we are segmenting\n",
+    "structure_list = [\n",
+    "    \"LUNG_L\",\n",
+    "    \"LUNG_R\",\n",
+    "    \"HEART\",\n",
+    "    \"SPINALCORD\",\n",
+    "    \"ESOPHAGUS\",\n",
+    "]\n",
+    "\n",
+    "# Define the atlas set\n",
+    "atlas_set = [\"101\", \"102\", \"103\", \"104\"]\n",
+    "\n",
+    "user_settings['atlas_settings'] = {\n",
+    "    'atlas_id_list': atlas_set,\n",
+    "    'atlas_structure_list': structure_list,\n",
+    "    'atlas_path': './data/nifti/lung',\n",
+    "    'atlas_image_format': 'LCTSC-Test-S1-{0}/IMAGES/LCTSC_TEST_S1_{0}_0_CT_0.nii.gz',\n",
+    "    'atlas_label_format': 'LCTSC-Test-S1-{0}/STRUCTURES/LCTSC_TEST_S1_{0}_0_RTSTRUCT_{1}.nii.gz',\n",
+    "    'crop_atlas_to_structures': True,\n",
+    "    'crop_atlas_expansion_mm': 10,\n",
+    "}\n",
+    "\n",
+    "# optionally, we can change some of the default registration parameters\n",
+    "user_settings[\"linear_registration_settings\"] = {\n",
+    "    \"reg_method\": \"similarity\",\n",
+    "    \"shrink_factors\": [8, 4, 2],\n",
+    "    \"smooth_sigmas\": [4, 2, 0],\n",
+    "    \"sampling_rate\": 1,\n",
+    "    \"default_value\": -1000,\n",
+    "    \"number_of_iterations\": 50,\n",
+    "    \"metric\": \"mean_squares\",\n",
+    "    \"optimiser\": \"gradient_descent\",\n",
+    "    \"verbose\": False\n",
+    "}\n",
+    "\n",
+    "user_settings[\"deformable_registration_settings\"] = {\n",
+    "    \"isotropic_resample\": True,\n",
+    "    \"resolution_staging\": [8],#,4,2],\n",
+    "    \"iteration_staging\": [50],#,50,25],\n",
```

Comment:
I don't think the optimiser will even get to 50 iterations at 8 mm resolution - so we could safely reduce this to something like 10. I suggest not worrying about it in this case, as it won't make much difference.